python/activator/activator.py

```diff
@@ -50,8 +51,15 @@
 timeout = os.environ.get("IMAGE_TIMEOUT", 50)
 # Absolute path on this worker's system where local repos may be created
 local_repos = os.environ.get("LOCAL_REPOS", "/tmp")
-
-setup_google_logger(
+# Kafka server
+kafka_cluster = os.environ["KAFKA_CLUSTER"]
+# Kafka group; must be worker-unique to keep workers from "stealing" messages for others.
+kafka_group_id = str(uuid.uuid4())
+# The topic on which to listen to updates to image_bucket
+# bucket_topic = f"{instrument_name}-image"
+bucket_topic = "pp-bucket-notify-topic"
```

Comment:
Should this be configurable (i.e. via env var)?

Reply:
I'm worried about how well that would work once we have topics that depend on the instrument, detector, etc.

Reply:
Instrument has to have a different set of workers, I would think.  You wouldn't be dynamically switching a given worker to a different instrument.

Detector/raft might be a problem.  At least the non-dynamic part should be configurable, then?  Or have the configuration specify a template?

Reply:
I guess I'm mostly thinking of a deployment per instrument and so having the instrument be part of the configuration and not the code is important.

Reply:
The instrument name itself is already part of the configuration -- we need it for lots of things!